### PR TITLE
Serialize arguments using Marshal

### PR DESCRIPTION
**Pull request by iadbbwarln**
Also other small fixes:
 * allow Process test run independently

### CODE REVIEW

**Comment by iadbbwarln** [comment]
CC @tomasv @oleg-vinted

File: lib/chained_job/store_job_arguments.rb [hunk 1]
```diff
@@ -44,6 +44,10 @@ def job_key
       @job_key ||= Helpers.job_key(job_class)
     end
 
+    def serialize(arguments)
+      arguments.map {|e| Marshal.dump(e)}
```
Comment (oleg-vinted) [inline lib/chained_job/store_job_arguments.rb:48]: * Do not use single-letter variables
* Surround block contents with spaces

File: test/chained_job/store_job_arguments_test.rb [hunk 2]
```diff
@@ -39,6 +39,10 @@ def job_tag
     current_time.to_f.to_s
   end
 
+  def stored_job_arguments
+    ARRAY_OF_JOB_ARGUMENTS.map {|e| Marshal.dump(e) }
```
Comment (oleg-vinted) [inline test/chained_job/store_job_arguments_test.rb:43]: Same here.

**Review by oleg-vinted — APPROVED** [review]
Minor style comments, otherwise :+1:

**Pull request merged**